OpenRA.Mods.Common/Traits/BotModules/Squads/States/NavyStates.cs

```diff
@@ -100,7 +101,17 @@ public void Tick(Squad owner)
 			if (!owner.IsValid)
 				return;
 
-			if (!owner.IsTargetValid)
+			// Check if we have an enemy naval yard this tick
+			var first = owner.Units.First();
+			var domainIndex = first.World.WorldActor.Trait<DomainIndex>();
+			var locomotorInfo = first.Info.TraitInfo<MobileInfo>().LocomotorInfo;
+			var navalProductions = owner.World.ActorsHavingTrait<Building>().Where(a
+			=> owner.SquadManager.Info.NavalProductionTypes.Contains(a.Info.Name)
+			&& domainIndex.IsPassable(first.Location, a.Location, locomotorInfo)
+			&& a.AppearsHostileTo(first));
```

Comment:
lines 109-111 should receive one mor level of indentation to make it more visible that they're par of `var navalProductions`.

Reply:
Will do.